### PR TITLE
Remove outdated requirements file

### PR DESCRIPTION
**Pull request by mzulqarnain1**
Relevant JIRA : https://openedx.atlassian.net/browse/BOM-1948

### CODE REVIEW

File: requirements/base.in [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # Core requirements for using this package
 -c constraints.txt
 
+django-pyfs>=1.0.5
```
Comment (jmbowman) [inline requirements/base.in:4]: Anything that depends on Django can't be in the core dependencies; the intention is for the default XBlock installation to not depend on Django, with dependencies for the optional Django support only being installed via `pip install XBlock[django]`.

Comment (mzulqarnain1) [reply requirements/base.in]: Removed from base.in, I see they are already present in installation requirements in `setup.py`

**Comment by jmbowman** [comment]
I guess I wasn't clear enough in the ticket; the problem isn't with `requirements/django.txt` (which we use to pin the version of Django used for testing the optional Django support, so tests don't suddenly break when a new version is released), but with `django_requirements.txt` in the repo's root directory (which I don't think anything uses anymore).  I think you can just delete that latter file without changing any of the others.

**Comment by mzulqarnain1** [comment]
> I guess I wasn't clear enough in the ticket; the problem isn't with `requirements/django.txt` (which we use to pin the version of Django used for testing the optional Django support, so tests don't suddenly break when a new version is released), but with `django_requirements.txt` in the repo's root directory (which I don't think anything uses anymore). I think you can just delete that latter file without changing any of the others.

@jmbowman Ohh. I think I misunderstood it from the ticket, sorry about that. You are totally right, there is no use of that file anywhere. I've removed that file.

**Pull request merged**